telco-hub/configuration/reference-crs/optional/odf-internal/odfReady.yaml

```diff
@@ -0,0 +1,73 @@
+---
+# this policy will ensure ODF is ready
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: odf-ready-validation
+  namespace: hub-policies
+  annotations:
+    # we do a validation of odf after StorageCluster creation
+    # but after MCH on ACM is created.
+    # MCH components like Observability and AgentServiceConfig
+    # come later and needs ODF ready
+    argocd.argoproj.io/sync-wave: "5" 
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    policy.open-cluster-management.io/description: Policy used ensure ODF installation is ready
+spec:
+  remediationAction: inform  # we dont want to modify/create only validate the status
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: odf-ready
+        spec:
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: batch/v1
+                kind: Job
+                metadata:
+                  namespace: openshift-storage
```

Comment:
This will look for any Job in the NS which meets the criteria. Is that the desire here? (is there only one Job?)

Reply:
there are some more. But I think it is oka to expect all were completed successfully.